roles/kubernetes_setup/files/scheduler-pod.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "kube-scheduler"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-scheduler",
+        "image": "registry.access.redhat.com/rhel7/kubernetes-scheduler",
```

Comment:
We could make this a selectable option so that we have the option of testing both containerized and in-sytstem kube.